core/controllers/feedback_updates.py

```diff
@@ -0,0 +1,222 @@
+from core.controllers import base
+from core.domain import exp_fetchers
+from core.domain import feedback_services
+from core.domain import subscription_services
+from core.domain import suggestion_registry
+from core.domain import suggestion_services
+from core.domain import user_services
+
+from typing import Dict, List, Optional, TypedDict, Union
+
+
+class MessageSummaryDict(TypedDict):
+    """Dict representation of author's messages summary."""
+
+    message_id: int
+    text: str
+    updated_status: str
+    author_username: Optional[str]
+    created_on_msecs: float
+
+
+class SuggestionSummaryDict(TypedDict):
+    """Dict representation of suggestion's summary."""
+
+    suggestion_html: str
+    current_content_html: str
+    description: str
+    author_username: Optional[str]
+    created_on_msecs: float
+
+
+class FeedbackUpdatesPage(
+    base.BaseHandler[Dict[str, str], Dict[str, str]]
+):
+    """Page showing the user's feedback updates."""
+
+    URL_PATH_ARGS_SCHEMAS: Dict[str, str] = {}
+    HANDLER_ARGS_SCHEMAS: Dict[str, Dict[str, str]] = {'GET': {}}
+
+    @acl_decorators.can_access_feedback_updates
+    def get(self) -> None:
+        """Handles GET requests."""
+        self.render_template('feedback-updates-page.mainpage.html')
+
+
+class FeedbackUpdatesHandlerNormalizedPayloadDict(TypedDict):
+    """Dict representation of FeedbackUpdatesHandler's
+    normalized_payload dictionary.
+    """
+
+    paginated_threads_list: List[List[str]]
+
+
+class FeedbackUpdatesHandler(
+    base.BaseHandler[
+        FeedbackUpdatesHandlerNormalizedPayloadDict,
+        Dict[str, str]
+    ]
+):
+    """Provides data for the user's feedback updates page."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS: Dict[str, str] = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'paginated_threads_list': {
+                'schema': {
+                    'type': 'list',
+                    'items': {
+                        'type': 'list',
+                        'items': {
+                            'type': 'basestring'
+                        },
+                    },
+                },
+                'default_value': []
+            }
+        }
+    }
+
+    @acl_decorators.can_access_feedback_updates
+    def post(self) -> None:
+        """Handles POST requests."""
+        assert self.user_id is not None
+        assert self.normalized_payload is not None
+        if len(self.normalized_payload['paginated_threads_list']) == 0:
+            full_thread_ids = (
+                subscription_services.get_all_threads_subscribed_to(
+                    self.user_id))
+            paginated_threads_list = [
+                full_thread_ids[index: index + 100]
+                for index in range(0, len(full_thread_ids), 100)]
+        else:
+            paginated_threads_list = self.normalized_payload[
+                'paginated_threads_list']
```

Comment:
```suggestion
            full_thread_ids = (
                subscription_services.get_all_threads_subscribed_to(
                    self.user_id
                )
            )
            paginated_threads_list = [
                full_thread_ids[index: index + 100]
                for index in range(0, len(full_thread_ids), 100)
            ]
        else:
            paginated_threads_list = self.normalized_payload[
                'paginated_threads_list'
            ]
```

Reply:
Also is 100 the size or something? If yes can we make it as a constant in this file?